noir-projects/noir-contracts/contracts/schnorr_account_contract/src/main.nr

```diff
@@ -92,4 +93,48 @@ contract SchnorrAccount {
         // docs:end:entrypoint
         true
     }
+
+    /**
+    * @notice Helper function to check the existing and validity of authwitnesses
+    * @dev TODO: myself and block_number should be removed and passed from a context
```

Comment:
Not directly as this one is doing both public and private, likely better addressed by having a unconstrained context as we briefly discussed the other day in the scrum. Should likely have a larger issue as it can also align functions so they are not totally different.